templates/serverpod_templates/projectname_server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM dart:2.19.0 AS build
+FROM dart:3 AS build
```

Comment:
We use 3.0 In our test docker containers. Should we use that here as well?

Reply:
Hmm maybe, I looked at dockerhub they specifically wrote 3 on the main page but maybe 3.0 is reasonable? Perhaps 3 will take the latest minor version?